po/fr.po

```diff
@@ -2034,17 +2035,19 @@ msgstr "Installation"
 #: src/cargo.md
 msgid "**Please follow the instructions on <https://rustup.rs/>.**"
 msgstr ""
+"**Veuillez suivre les instructions sur <https://rustup.rs/>** (en anglais)."
 
 #: src/cargo.md
-#, fuzzy
 msgid ""
 "This will give you the Cargo build tool (`cargo`) and the Rust compiler "
 "(`rustc`). You will also get `rustup`, a command line utility that you can "
 "use to install to different compiler versions."
 msgstr ""
-"Avec cargo et rustc, Rustup s'installera en tant qu'utilitaire de ligne de "
-"commande que vous pourrez utiliser pour installer/changer de chaînes "
-"d'outils, configurer une compilation croisée, etc."
+"Une fois que vous aurez suivi ces instructions, vous disposerez du "
+"gestionnaire de dépendances et système de compilation `cargo` et du "
+"compilateur `rustc`. `rustup` sera également installé : il s'agit d'un "
+"utilitaire en ligne de commande que vous pourrez utiliser pour installer/"
+"changer de chaînes d'outils, configurer une compilation croisée, etc."
```

Comment:
**Suggestion:** Translation changes the clarity of information.

The translation changes the meaning of the sentence. In my opinion, the original sentence is more concise and less confusing for the reader, so it seems wiser to leave it unchanged.

```suggestion
"utilitaire en ligne de commande que vous pourrez utiliser pour installer "
"différentes versions du compilateur."
```